agent/ecscni/plugin_test.go

```diff
@@ -259,3 +263,31 @@ func TestCNIPluginVersion(t *testing.T) {
 		})
 	}
 }
+
+// Returns the version in CNI plugin VERSION file as a string
+func getCNIVersionString(t *testing.T) string {
```

Comment:
nit: we prefer having caller function above callee, so that it's easier to read.